### ISSUE TITLE
Bug: `craetesuperuser` doesn't work

### ISSUE THREAD

**Issue by witoszekdev** [saleor/saleor#11753]
### What are you trying to achieve?

I'm trying to create new superuser by using the `./manage.py createsuperuser` command.

### Steps to reproduce the problem

1. Clone [saleor-platform](https://github.com/saleor/saleor-platform)
2. Checkout `saleor` to `main` branch
3. Build docker images: `docker compose build api`
4. Start Saleor: `docker compose -d up api worker`
5. Run migrations: `docker-compose run --rm api python3 manage.py migrate`
6. Populate database: `docker-compose run --rm api python3 manage.py populatedb`
7. Run `createsuperuser` command: `docker-compose run --rm api python3 manage.py createsuperuser`

### What did you expect to happen?

I expected an interactive input to provide email and password for new superuser account.

The create account should have all permissions in Saleor, just like `populatedb --createsuperuser` admin account.

### Logs

```
2023-01-18 14:30:19,333 WARNING saleor.core.jwt_manager RSA_PRIVATE_KEY is missing. Using temporary key for local development with DEBUG mode. [PID:1:MainThread]
Traceback (most recent call last):
  File "/app/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/management/commands/createsuperuser.py", line 79, in execute
    return super().execute(*args, **options)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/management/commands/createsuperuser.py", line 100, in handle
    default_username = get_default_username(database=database)
  File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/management/__init__.py", line 115, in get_default_username
    from django.contrib.auth import models as auth_app
  File "/usr/local/lib/python3.9/site-packages/django/contrib/auth/models.py", line 35, in <module>
    class Permission(models.Model):
  File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 113, in __new__
    raise RuntimeError(
RuntimeError: Model class django.contrib.auth.models.Permission doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
ERROR: 1
```

### Environment

- Saleor version: latest (`4d5b30419f7a7bb9aa572eb2e085ab7277a68601`)
- OS and version: macOS 13.0.1
- Docker version 20.10.21, build baeda1f
- docker-compose version 1.29.2, build 5becea4c
- Saleor platform version: latest (`4d5b30419f7a7bb9aa572eb2e085ab7277a68601`)

**Comment by james-vervitas** [comment]
I was having the same issue. 

I used the command found [here](https://github.com/saleor/saleor-platform/blob/main/README.md#troubleshooting:~:text=Populate%20the%20database%20with%20example%20data%20and%20create%20the%20admin%20user%3A) instead, and did not return an error

`docker-compose run --rm api python3 manage.py populatedb --createsuperuser`

This allowed me to use admin@example.com with the password set to admin

**Comment by ellicodan** [comment]
> I was having the same issue.
> 
> I used the command found [here](https://github.com/saleor/saleor-platform/blob/main/README.md#troubleshooting:~:text=Populate%20the%20database%20with%20example%20data%20and%20create%20the%20admin%20user%3A) instead, and did not return an error
> 
> `docker-compose run --rm api python3 manage.py populatedb --createsuperuser`
> 
> This allowed me to use [admin@example.com](mailto:admin@example.com) with the password admin

This fixed it for me, thank you. My guess is that the populatedb command is not optional, as the docs suggested?

**Comment by romeopeter** [comment]
I'm having the same issue but locally on my Mac without running docker. How do I fix this?

**Comment by fowczarek** [comment]
Hello @romeopeter, 
Which Saleor version do you use? (It was released as patch 3.10.3, and should work correctly.)
Meantime, you could use the workaround described above by @james-vervitas
